Sources/Models/Views/ScrollViews/CollectionView.swift

```diff
@@ -82,50 +97,74 @@ public struct CollectionViewCell: XMLDecodable, ViewProtocol {
-            userDefinedRuntimeAttributes:              xml.byKey("userDefinedRuntimeAttributes")?.children.flatMap(decodeValue),
-            connections:                               xml.byKey("connections")?.children.flatMap(decodeValue)
+            id:                                        try container.attribute(of: .id),
+            autoresizingMask:                          container.elementIfPresent(of: .autoresizingMask),
+            clipsSubviews:                             container.attributeIfPresent(of: .clipsSubviews),
+            constraints:                               constraintsContainer?.elementsIfPresent(of: .constraint),
+            contentView:                               try container.element(of: .contentView),
+            contentMode:                               container.attributeIfPresent(of: .contentMode),
+            customClass:                               container.attributeIfPresent(of: .customClass),
+            customModule:                              container.attributeIfPresent(of: .customModule),
+            isMisplaced:                               container.attributeIfPresent(of: .isMisplaced),
+            opaque:                                    container.attributeIfPresent(of: .opaque),
+            rect:                                      try container.element(of: .rect),
+            _subviews:                                 container.childrenIfPresent(of: ._subviews),
+            translatesAutoresizingMaskIntoConstraints: container.attributeIfPresent(of: .translatesAutoresizingMaskIntoConstraints),
+            userInteractionEnabled:                    container.attributeIfPresent(of: .userInteractionEnabled),
+            userDefinedRuntimeAttributes:              container.childrenIfPresent(of: .userDefinedRuntimeAttributes),
+            connections:                               container.childrenIfPresent(of: .connections)
         )
     }
 }
 
 // MARK: - CollectionViewLayout
 
-public struct CollectionViewLayout: XMLDecodable {
+public struct CollectionViewLayout: XMLDecodable, KeyDecodable {
 
     public let id: String
     public let key: String?
     public let customClass: String?
     public let customModule: String?
 
     static func decode(_ xml: XMLIndexer) throws -> CollectionViewLayout {
+        let container = xml.container(keys: MappedCodingKey.self).map { (key: CodingKeys) in
+            let stringValue: String = {
+                switch key {
+                case .key: return "contentMode"
+                default: return key.stringValue
+                }
+            }()
+            return MappedCodingKey(stringValue: stringValue)
+        }
         return CollectionViewLayout(
-            id:                       try xml.attributeValue(of: "id"),
-            key:                      xml.attributeValue(of: "contentMode"),
-            customClass:              xml.attributeValue(of: "customClass"),
```

Comment:
I think I make an issue here. key is key , no mapping needed
#22

Reply:
@phimage So what do you suggest, using a constant like `"collectionViewLayout"`?
Or do you mean the typo for `contentMode -> key`? Cuz it's been fixed

Reply:
If I fix my code
```swift
key:                      xml.attributeValue(of: "key")
```
so `.key`will be used without key mapping